packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.types.ts

```diff
@@ -137,6 +147,12 @@ export interface IColorPickerStrings {
    */
   alpha?: string;
 
+  /**
+  Label for the transparency text field.
+  * @defaultvalue Transparency
+  */
+  transparency?: string;
```

Comment:
I think you should not have a separate prop for transparency label here as well as above. We can repurpose alpha label as transparency label. Or Alternatively, you should also take transparencyAriaLabel as an input

Reply:
I could go either way on this. The benefit of a separate label is that it's clearer how to properly make a transparency slider, whereas the benefit of reusing the alpha label is that it's a smaller API surface overall.

If you keep the separate label, you should add a comment that it's only used when the certain prop (whatever we decide to call it) is true. Or if you combine the labels, you should update the comments on `a` and `alphaLabel` to indicate that they're also used for transparency when the prop is true.

Reply:
If you do keep the separate `transparency` label, you should add an additional one `transparencyAriaLabel` in case anyone would like to set the aria label differently from the visual label.

Reply:
re-purposing alpha label for transparency as well now!!

Reply:
@ecraig12345 i have added separate label "transparency" and separate aria label "transparencyAriaLabel" for better clarity.